### ISSUE TITLE
Adjustments to the [1-Meaning, All Cognate Sets] form

### ISSUE THREAD

**Issue by CasFre** [lingdb/CoBL#201]
- [ ] Narrow the columns of colour block = - [ ] Reduce space taken up by colored boxes

@runjak: Could you re-arrange next week the one mean - all cognate sets table so that I do see the complete root forms again?
Of course, my screen isn't that big, but obviously that is not the problem, for, as you can see in the screenshot, the display of the website does not adjust automatically to screen size, for there is the scroll bar at the bottom of the page.

@cormacanderson and @PaulHeggarty might also want to have the lexeme counts moved to the left edge, i.e. directly after the colour boxes...

**Comment by CasFre** [comment]
![root form fields](https://cloud.githubusercontent.com/assets/17431124/16267243/0604c9be-3889-11e6-807a-428634a9f02d.jpg)

**Comment by PaulHeggarty** [comment]
For a start I suggest narrowing the colour boxes a bit.
And now that you've got the vertical text in column labels working, you could use if for the count columns, for example, and all the boolean check boxes, to save some space.  This should be done generally across various forms. It's particularly critical on the languages reference table too, where we can’t even see the full language names on a wide screen.

**Comment by cormacanderson** [comment]
As far as I can see, this issue has been resolved, so can now be closed. This tracker issue https://github.com/lingdb/CoBL/issues/231 can probably be used for such requests in the future.

**Comment by PaulHeggarty** [comment]
The issue has been to a good extent alleviated by a series of adjustments during a 'mobbing' on Tuesday 2016 07 26.  The check box in the header would help further, but has not yet been implemented because it is not a simple solution from a programming perspective, and would take a lot of Jakob's time, so its priority was downgraded, but it is still desirable and not yet implemented.

**Comment by cormacanderson** [comment]
Fair enough. @runjak could you also please put one of the light vertical bars between "Dubious Set" and "Loan Event" in this html form, and change the spelling of "Idiophonic" to "Ideophonic".

**Comment by PaulHeggarty** [comment]
These are immediate quick things, so I will move them to #263.  
Also, requests like this should always, if possible, go up as new check-box points in the first entry in an issue, its description text, so that Jakob knows when he has finished all the requests, and we to can see the progress count within the general issue list, even without having to open the issue.
